create-table.md

```diff
@@ -222,6 +222,28 @@ We also have other resources on indexes:
 - Create indexes for existing tables using [`CREATE INDEX`](create-index.html).
 - [Learn more about indexes](indexes.html).
 
+### Create a Table with Auto-Generated Row IDs
+
+To auto-generate unique row IDs, use an [`INT`](int.html) column with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the column default:
+
+~~~ sql
+> CREATE TABLE test (
+    id INT PRIMARY KEY DEFAULT unique_rowid(), 
+    name STRING
+);
+~~~
+
+On insert, the `unique_rowid()` function will generate a default value from the insert timestamp and the ID of the node executing the insert, a combination that is guaranteed to be globally unique. 
+
+Alternately, you can use the CockroachDB-specific [`SERIAL`](serial.html) data type, which is simply an alias for the `INT` usage above:
```

Comment:
`SERIAL` is also used by PostgreSQL (and that's why we use this name). We should acknowledge this, and also talk about how our SERIAL is different from PostgreSQL's SERIAL.